### PR TITLE
Path Component Accessible label

### PR DESCRIPTION
**Pull request by davidjray**
Adding accessible label to Path component

Note the test snapshots are auto-generated by adding the `--updateSnapshot` flag when running JS tests.

### CODE REVIEW

**Comment by davidjray** [comment]
Epic screen example:
Screen Reader before change
Screen Shot 2022-10-18 at 10 09 12 AM

After Change
Screen Shot 2022-10-18 at 10 10 31 AM

File: locales_dev/en/translation.json [hunk 1]
```diff
@@ -143,6 +143,7 @@
   "Epic Name": "Epic Name",
   "Epic Options": "Epic Options",
   "Epic Scratch Org": "Epic Scratch Org",
+  "Epic Status": "Epic Status",
```
Comment (davidmreed) [inline locales_dev/en/translation.json:146]: I believe these new strings need to be copied over to `locales/en/translation.json` to be included in the next localization pass.

File: src/js/components/path/index.tsx [hunk 2]
```diff
@@ -7,16 +7,18 @@ interface PathProps {
...
-const Path = ({ steps, activeIdx, isCompleted, isLost }: PathProps) => (
+const Path = ({ steps, activeIdx, isCompleted, isLost, label }: PathProps) => (
   <div className="slds-region_small">
     <div className="slds-path">
       <div className="slds-grid slds-path__track">
         <ul
           className="slds-path__nav"
           role="listbox"
           aria-orientation="horizontal"
+          aria-label={label}
```
Comment (davidmreed) [inline src/js/components/path/index.tsx:21]: Is an empty string a valid value for `aria-label`? Just noting that `label` is nullable - do we need to worry about that?

**Review by davidmreed — CHANGES_REQUESTED** [review]

**Review by davidmreed — APPROVED** [review]

File: src/js/components/path/index.tsx (hunk 2 above)
Comment (davidjray) [reply src/js/components/path/index.tsx:21]: Empty `aria-labels` are ignored, but great question!

**Pull request merged**